backend/app/requests/representative_points.py

```diff
@@ -51,11 +51,12 @@ def representative_points_request(app, flask_request, engine):
     try:
         request_json = flask_request.get_json(force=True)
         service_area_ids = request_json['service_area_ids']
+        census_data = request_json.get('census_data', False) and config.get('include_census_data')
```

Comment:
What does `include_census_data` indicate? Is it talking about RPs? The database?